app/serializers/v1/GroupSerializer.js

```diff
@@ -2,6 +2,6 @@ var Serializer = require("../../models").Serializer
 
 exports.addSerializer = function() {
   return new Serializer('groups', {
-    select: ['id', 'username', 'type', 'screenName']
+    select: ['id', 'username', 'type', 'screenName', 'visibility', 'postsTimelineId']
```

Comment:
I also think that specifying group names when posting is a better design. I was simply sticking to the design of the 'master' branch; I'll change it if you also think that specifying group names is better.